docker/Dockerfile.build.aarch64-qnx

```diff
@@ -91,6 +91,13 @@ RUN JPEG_TURBO_VERSION=2.0.5 && \
     make -j"$(grep ^processor /proc/cpuinfo | wc -l)" install 2>&1 >/dev/null && \
     rm -rf /libjpeg-turbo-${JPEG_TURBO_VERSION}
 
+# zstandard compression library
+RUN ZSTANDARD_VERSION=1.4.5 && \
+    curl -L https://github.com/facebook/zstd/releases/download/v${ZSTANDARD_VERSION}/zstd-${ZSTANDARD_VERSION}.tar.gz | tar -xzf - && \
+    cd zstd-${ZSTANDARD_VERSION} && \
+    make -j"$(grep ^processor /proc/cpuinfo | wc -l)" install 2>&1 >/dev/null && \
```

Comment:
You need to cross compile it for aarch64, see how other libraries are build.

Reply:
I was taking a look at QNX, I didn't even manage to understand which toolchain is good for Linux. Any pointer? I did some programming on embedded platforms in the past but never with QNX.

Reply:
I guess it should be something like this:
```
CFLAGS="-fPIC" \
    CXXFLAGS="-fPIC" \
    CC=aarch64-unknown-nto-qnx7.0.0-gcc \
    CXX=aarch64-unknown-nto-qnx7.0.0-g+ \
    prefix=/usr/aarch64-unknown-nto-qnx/aarch64l \
    make install -j"$(grep ^processor /proc/cpuinfo | wc -l)" && \
```
And similar (check how other deps are build there) for aarch64.

Reply:
And regarding the aarch64 for Nvidia Jetson, the SDKManager doesn't allow the download for ubuntu 20.04 :-(

Reply:
Sorry, I cannot do much about it.

Reply:
I'm trying to get a temp license for qnx and test the compilation on that platform. Regarding the aarch64 for nvidia jetson, I will see in the coming days to use a box with ubuntu 18.04 installed

Reply:
Great!.
Thanks.

Reply:
So I managed to install on my host (ubuntu 20.04) qnx 7.1 and I tried to compile the zstd library. As I cannot use the SDKManager I could not test the docker build but I think the modifications I did in the last commits should work. On my host the zstd library is correctly compiled and linked

```bash
lib file libzstd.so.1.4.6 
libzstd.so.1.4.6: ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, BuildID[md5/uuid]=e96f8cf465e8cec436c1487611392dc9, with debug_info, not stripped
```

Reply:
Sure, let me try it on our CI.